### PR TITLE
FEATURE: LEAF-2405 - Dedicated Session Timeout Page

### PR DESCRIPTION
**Pull request by gsteinhoff**

### CODE REVIEW

File: LEAF_Request_Portal/session_expire.php [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<?php
+// Include XSSHelpers
+if (!class_exists('XSSHelpers'))
+{
+    include_once __DIR__ . '/../libs/php-commons/XSSHelpers.php';
+}
+
+include '../libs/smarty/Smarty.class.php';
+
+function customTemplate($tpl)
```
Comment (Pelentan) [inline LEAF_Request_Portal/session_expire.php:10]: Need default value for $tpl attribute.  Additionally type hinting.  https://www.php.net/manual/en/language.types.declarations.php.  Would like to start seeing this both for the arguments and return type.  Exp:

function customTemplate(string $tpl='default_tpl_name')  ?string { ... }

File: LEAF_Request_Portal/templates/view_session_expire.tpl [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        </div>
+        <span id="headerTab"></span>
+        <span id="headerTabImg"><img src="images/tab.png" alt="tab" /></span>
+    </div>
+    <div id="body">
+        <div id="content">
+            <div id="bodyarea">
+                <div class="card" style="width: 50%; margin: auto; padding: 16px">
+                    <h2>Your Session Has Expired</h2>
+                    <p>Return to your <a href="{$previousPage}">LEAF site</a></p>
```
Comment (Pelentan) [inline LEAF_Request_Portal/templates/view_session_expire.tpl:25]: Should this route to the "/login" page?  I don't think so, but probably ought to be asked.

**Review by Pelentan — CHANGES_REQUESTED** [review]

File: LEAF_Request_Portal/session_expire.php (hunk 1 above)
Comment (gsteinhoff) [reply LEAF_Request_Portal/session_expire.php]: This has been updated in the latest commit.

File: LEAF_Request_Portal/templates/view_session_expire.tpl (hunk 2 above)
Comment (gsteinhoff) [reply LEAF_Request_Portal/templates/view_session_expire.tpl]: This has been changed in the latest commit as not really need for the previous page if next page should be login anyway, not back to the old page.

**Review by pete-nerantzinis — APPROVED** [review]
Looks good

**Review by raphaelfretter — APPROVED** [review]
looks good

**Review by maxrichardjr — APPROVED** [review]
Thanks for the IDs!

**Review by AmandaSumner — APPROVED** [review]

**Pull request merged**